WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift

```diff
@@ -160,6 +162,8 @@ class NewGutenbergViewController: UIViewController, PostEditor, PublishingEditor
             content: post.content ?? "",
             service: GutenbergKit.EditorService(client: networkClient),
             themeStyles: FeatureFlag.newGutenbergThemeStyles.enabled,
+            plugins: FeatureFlag.newGutenbergPlugins.enabled && isSelfHosted,
```

Comment:
The remote editor is limited to self-hosted sites currently as it sites authenticating with a WPCOM account utilize the WPCOM API and an OAuth token. The remote editor requires a third-party endpoint found on the core WP API and an application password token (p1725915009777259/1725894719.476849-slack-C04PWEZSYFL).